client/src/components/AdminCard.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Avatar } from './Avatar';
+import PropTypes from 'prop-types';
+
+export const AdminCard = ({ user }) => {
+  return (
+    <>
```

Comment:
Generally I think you add in some container - when rendered in a list for example, it will appear as a series of span/avatar pairs, which won't look natural in the DOM view of this.